operations/agent.go

```diff
@@ -20,46 +20,21 @@ import (
 
 func Agent() cli.Command {
 	const (
-		hostIDFlagName           = "host_id"
-		hostSecretFlagName       = "host_secret"
-		apiServerFlagName        = "api_server"
-		workingDirectoryFlagName = "working_directory"
-		logPrefixFlagName        = "log_prefix"
-		statusPortFlagName       = "status_port"
-		cleanupFlagName          = "cleanup"
-		logkeeperFlagName        = "logkeeper_url"
-		s3BaseFlagName           = "s3_base_url"
+		logPrefixFlagName  = "log_prefix"
+		statusPortFlagName = "status_port"
+		cleanupFlagName    = "cleanup"
+		s3BaseFlagName     = "s3_base_url"
 	)
 
 	return cli.Command{
 		Name:  "agent",
 		Usage: "run an evergreen agent",
-		Flags: []cli.Flag{
-			cli.StringFlag{
-				Name:  hostIDFlagName,
-				Usage: "id of machine agent is running on",
-			},
-			cli.StringFlag{
-				Name:  hostSecretFlagName,
-				Usage: "secret for the current host",
-			},
-			cli.StringFlag{
-				Name:  apiServerFlagName,
-				Usage: "URL of the API server",
-			},
-			cli.StringFlag{
-				Name:  workingDirectoryFlagName,
-				Usage: "working directory for the agent",
-			},
+		Flags: append(addCommonAgentAndRunnerFlags(),
```

Comment:
I would omit the append and just put the extra flags as args to the addCommonAgentAndRunnerFlags()